CalcualtorLogic.js

```diff
@@ -34,6 +34,7 @@ class Calculator {
         wrapperDiv.classList.add("wrapper");
         resultDiv.classList.add("result");
         resultDiv.id = "result";
+        document.wrapperDiv.appendChild(resultDiv);
```

Comment:
You don't need `document.` here 
`wrapperDiv.appendChild(resultDiv);` is enough 

in this example
```html
let p = document.createElement("p");
document.body.appendChild(p);
```
`document` is used to access the `body` tag.
one of the way to select the `body` tag is to use `document.body`